.eslintrc.js

```diff
@@ -2,12 +2,15 @@
 
 module.exports = {
   root: true,
-  parser: 'babel-eslint',
+  parser: '@babel/eslint-parser',
   parserOptions: {
-    ecmaVersion: 2018,
+    ecmaVersion: 'latest',
```

Comment:
is latest fine? I read a whole discussion about it but I guess it's fine?
https://github.com/eslint/eslint/issues/7528

Reply:
I would guess this is totally fine (famous last words 😅)

Reply:
suits me